src/renderer/Button.tsx

```diff
@@ -1,23 +1,25 @@
-import './styles/LargeButton.scss';
+import './styles/Button.scss';
 
-interface ILargeButtonProps {
+interface IButtonProps {
   children: JSX.Element;
   ariaLabel: string;
   isDisabled: boolean;
+  className?: string;
   handleChange: () => void;
 }
 
-const LargeButton = ({
+const Button = ({
   children,
   ariaLabel,
   isDisabled,
+  className = 'button',
   handleChange,
-}: ILargeButtonProps) => {
+}: IButtonProps) => {
   return (
     <div
       role="button"
       aria-label={ariaLabel}
-      className="button"
+      className={className}
```

Comment:
super small thing that can be done later - we could add a `type` prop that takes 'small' or 'large' (with a default set to one or the other? idk) that adds the `small` class depending on the type. This way we can easily toggle between the small and large button variants